src/Uno.UI/DataBinding/BindingPropertyHelper.FastConvert.cs

```diff
@@ -334,6 +339,240 @@ private static bool FastStringConvert(Type outputType, string input, ref object
 			return false;
 		}
 
+		private static bool FastStringToNumericConvert(Type outputType, string input, ref object output)
+		{
+			const NumberStyles numberStyles = NumberStyles.AllowDecimalPoint | NumberStyles.AllowLeadingSign | NumberStyles.Float | NumberStyles.AllowLeadingWhite | NumberStyles.AllowTrailingWhite;
+
+			if (outputType == typeof(double))
```

Comment:
split this into different methods, there's also already a FastStringToDouble converter below.